### PR TITLE
added Every Single Miscellaneous Block

### PR DESCRIPTION
**Pull request by NemEzanevem**
## What
Adding the Remaining missing Blocks to Modern from 1.12

## Implementation Details
-

## Outcome
-

## Additional Information
accidentally pulled some older research stuff in the commit...

## Potential Compatibility Issues
-

### CODE REVIEW

**Comment by screret** [comment]
nem you already have conflicts.

File: src/main/java/com/gregtechceu/gtceu/data/recipe/CustomTags.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public class CustomTags {
 
 
     // Platform-dependent tags
-    public static final TagKey<Item> TAG_WOODEN_CHESTS = TagUtil.createItemTag("chests/wooden");
+    public static final TagKey<Item> RUBBER_LOGS_ITEM = TagUtil.createModItemTag("chests/wooden");
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/data/recipe/CustomTags.java]: this is supposed to be a forge tag.

Comment (screret) [reply src/main/java/com/gregtechceu/gtceu/data/recipe/CustomTags.java]: wait why did you call it `RUBBER_LOGS_ITEM`

File: src/main/java/com/gregtechceu/gtceu/data/recipe/CustomTags.java [hunk 2]
```diff
@@ -86,6 +86,9 @@ public class CustomTags {
     public static final TagKey<Block> CREATE_SEATS = TagUtil.optionalTag(Registries.BLOCK, new ResourceLocation(GTValues.MODID_CREATE, "seats"));
     public static final TagKey<Block> ORE_BLOCKS = TagUtil.createBlockTag("ores");
 
+    public static final TagKey<Block> RUBBER_LOGS_BLOCK = TagUtil.createModBlockTag("logs/rubber");
+
+    public static final TagKey<Item> WOODEN_CHESTS = TagUtil.createModItemTag("chests/wooden");
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/data/recipe/CustomTags.java]: why mod?

File: src/main/resources/assets/gtceu/textures/armor/rubber_door_top.png
Comment (screret) [inline src/main/resources/assets/gtceu/textures/armor/rubber_door_top.png:1]: allll of these are also corrupt.

Comment (screret) [reply src/main/resources/assets/gtceu/textures/armor/rubber_door_top.png:1]: at least according to github.

File: src/main/java/com/gregtechceu/gtceu/data/recipe/misc/MetaTileEntityLoader.java [hunk 3]
```diff
@@ -325,8 +325,8 @@ public static void init(Consumer<FinishedRecipe> provider) {
 
         registerMachineRecipe(provider, GTMachines.FISHER, "WTW", "PMP", "TGT", 'M', HULL, 'W', CIRCUIT, 'G', PUMP, 'T', MOTOR, 'P', PISTON);
         registerMachineRecipe(provider, GTMachines.ITEM_COLLECTOR, "MRM", "RHR", "CWC", 'M', MOTOR, 'R', ROTOR, 'H', HULL, 'C', CIRCUIT, 'W', CABLE);
-        registerMachineRecipe(provider, GTMachines.BLOCK_BREAKER, "MGM", "CHC", "WSW", 'M', MOTOR, 'H', HULL, 'C', CIRCUIT, 'W', CABLE, 'S', CustomTags.TAG_WOODEN_CHESTS, 'G', GRINDER);
         if(ConfigHolder.INSTANCE.machines.enableWorldAccelerators)
+            registerMachineRecipe(provider, GTMachines.BLOCK_BREAKER, "MGM", "CHC", "WSW", 'M', MOTOR, 'H', HULL, 'C', CIRCUIT, 'W', CABLE, 'S', CustomTags.WOODEN_CHESTS, 'G', GRINDER);
```
Comment (mikerooni) [inline src/main/java/com/gregtechceu/gtceu/data/recipe/misc/MetaTileEntityLoader.java]: This shouldn't have been moved here

**Review by screret — CHANGES_REQUESTED** [review]

**Pull request closed without merging**